### PR TITLE
Docs: Update READMEs

### PR DESCRIPTION
**Pull request by robertjdominguez**
Updates READMEs to include the new `ddn connector init -i` workflow. Additionally, it standarizes the READMEs by reducing the amount of updates needed to CLI commands and — instead — links out to the docs. For any README that is of a "special child" (e.g., PostgreSQL), we've linked to the docs page(s).

### CODE REVIEW

File: registry/hasura/elasticsearch/README.md [hunk 1]
```diff
@@ -41,40 +46,28 @@ Below, you'll find a matrix of all supported features for the Elasticsearch conn
...
 
-## Using the Elasticsearch connector
-
-### Step 1: Authenticate your CLI session
-
-```bash
-ddn auth login
-```
+The steps below explain how to initialize and configure a connector for local development. You can learn how to deploy a
+connector — after it's been configured —
```
Comment (rsd1122) [inline registry/hasura/elasticsearch/README.md:58]: deploy a connector to DDN

File: registry/hasura/elasticsearch/README.md [hunk 2]
```diff
@@ -41,40 +46,28 @@ Below, you'll find a matrix of all supported features for the Elasticsearch conn
...
 4. [Create a subgraph](https://hasura.io/docs/3.0/getting-started/init-subgraph)
 
-## Using the Elasticsearch connector
-
-### Step 1: Authenticate your CLI session
-
-```bash
-ddn auth login
-```
+The steps below explain how to initialize and configure a connector for local development. You can learn how to deploy a
```
Comment (rsd1122) [inline registry/hasura/elasticsearch/README.md:57]: .. and configure a connector on your local machine (typically for development purposes).

**Review by rsd1122 — COMMENTED** [review]
Suggested a couple of changes to the Elasticsearch connector that are also applicable to other connectors as the terms "local development" and "deploy a connector" may not be clearly understood by everyone.

File: registry/hasura/elasticsearch/README.md [hunk 3]
```diff
@@ -41,40 +46,28 @@ Below, you'll find a matrix of all supported features for the Elasticsearch conn
...
 4. [Create a subgraph](https://hasura.io/docs/3.0/getting-started/init-subgraph)
 
-## Using the Elasticsearch connector
-
-### Step 1: Authenticate your CLI session
-
-```bash
-ddn auth login
-```
+The steps below explain how to initialize and configure a connector on your local machine (typically for development
```
Comment (gneeri) [inline registry/hasura/elasticsearch/README.md:57]: @robertjdominguez, I think the intent of these instructions was for cloud deployment of the project, not of the connector—i.e., the way I read this, is that after your done with your local installation of elastic, here's how you publish it to the cloud. @rsd1122, correct me if I'm mistaken, but if that is the intent, I think we should change the work `connector` below to `project` and change the link to the following: https://hasura.io/docs/3.0/getting-started/deployment/deploy-your-supergraph.

Comment (robertjdominguez) [reply registry/hasura/elasticsearch/README.md:57]: > I think the intent of these instructions was for cloud deployment of the project, not of the connector—i.e., the way I read this, is that after your done with your local installation of elastic, here's how you publish it to the cloud

I understood this as helping a user understand how to get started with their connector for local development. Then, once they've sorted their connection and are happy with what's generated in the local project, they can create a build on Hasura DDN.

To me, it's "how to configure a connector on your local machine [for use with your local project]."

Is that not what we're trying to communicate?

Comment (gneeri) [reply registry/hasura/elasticsearch/README.md:57]: Sorry @robertjdominguez, I stand corrected. What you had was indeed the correct link.

**Review by gneeri — APPROVED** [review]

**Pull request merged**